docs/aqua_migration.rst

```diff
@@ -0,0 +1,13 @@
+#####################
```

Comment:
Can we insert a reference point here so I can update the terra release notes to link to this.

Something like:

```suggestion
.. _aqua-migration:

#####################
```

Then in the terra release notes I can do `:ref:`aqua-migration` and have it link to this